apps/codeforafrica/src/payload/collections/Projects.js

```diff
@@ -0,0 +1,150 @@
+import image from "../fields/image";
+import linkArray from "../fields/links/linkArray";
+import linkGroup from "../fields/links/linkGroup";
+import richText from "../fields/richText";
+import slug from "../fields/slug";
+import tags from "../fields/tags";
+import nestCollectionUnderPage from "../utils/nestCollectionUnderPage";
+
+const Projects = {
+  slug: "projects",
+  admin: {
+    useAsTitle: "name",
+  },
+  fields: [
+    {
+      name: "name",
+      label: {
+        en: "Name",
+      },
+      type: "text",
+      required: true,
+    },
+    image({
+      overrides: {
+        label: {
+          en: "Icon",
+        },
+        name: "icon",
+        required: true,
+      },
+    }),
+    richText({
+      name: "title",
+      label: { en: "Title" },
+    }),
+    richText({
+      name: "subtitle",
+    }),
+    {
+      name: "tagLine",
+      label: {
+        en: "Tagline",
+      },
+      type: "text",
+    },
+    tags({
+      name: "tag",
+      hasMany: false,
+      admin: {
+        position: "sidebar",
+      },
+    }),
+    linkGroup({ overrides: { name: "externalLink", label: "Link" } }),
+    image({
+      overrides: {
+        label: {
+          en: "Thumbnail",
+        },
+        name: "thumbnail",
+        required: true,
+      },
+    }),
+    {
+      name: "badges",
+      type: "array",
+      label: {
+        en: "Badges",
+      },
+      fields: [
+        {
+          name: "name",
+          label: {
+            en: "Name",
+          },
+          type: "text",
+          required: true,
+        },
+        {
+          name: "date",
+          type: "date",
+          required: true,
+          admin: {
+            date: {
+              pickerAppearance: "dayAndTime",
+            },
+          },
+        },
+      ],
+    },
+    richText({
+      name: "description",
+    }),
+    linkArray({
+      linkConfig: {
+        defaultValue: "custom",
+        disableLinkTypeSelection: true,
+        labelFields: [
+          {
+            name: "iconSlug",
+            label: { en: "Icon Slug" },
+            type: "select",
+            options: [
+              {
+                value: "github",
+                label: { en: "Github Icon" },
+              },
+              {
+                value: "data",
+                label: { en: "Data Icon" },
+              },
+            ],
+          },
+        ],
+      },
+    }),
```

Comment:
This is hard to understand. Why set link icon?

I think the better approach is to have link type and we only allow one of two types: source code link (which we'll show using GitHub icon) and data link (which we'll show using database icon).

Otherwise, we might as well allow content editors to set actual icons.